### DISCUSSION TITLE
Failed to gRPC request [Tflint]

### DISCUSSION THREAD

**Discussion by RohanUB121** [terraform-linters/tflint#1742] [Q&A] [answered]
Tflint throws an error as shown in the below screenshot but tflint stage passes has anyone faced this issue before? any help is appreciated.  

<img width="1771" alt="image" src="https://user-images.githubusercontent.com/73757263/231230333-368d6e82-1077-4eb1-a31f-b18fdd026679.png">

**Comment by wata727** [comment, marked as answer]
This is an internal error that occurs when evaluating a variable whose value is unknown. This error is worth noting if you want to evaluate everything as a known value.

This is not an issue if you allow unknown values to be included. By the way, TFLint v0.46 and plugins built with SDK v0.16+ do not log this error.
